site.yml

```diff
@@ -0,0 +1,25 @@
+runtime:
+  cache_dir: ./.cache/antora
+site:
+  title: AeroGear Mobile Security
+  url: https://www.aerogear.org
+  start_page: mobile-security::index
+content:
+  sources:
+  - url: https://github.com/finp/mobile-security
```

Comment:
Is this file necessary for the shipped version or is it for building locally?

Reply:
will remove